google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointImplicitUsageProviderTest.java

```diff
@@ -15,29 +15,69 @@
  */
 package com.google.cloud.tools.intellij.appengine.validation;
 
+import com.intellij.openapi.util.io.FileUtil;
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiMethod;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+
 /**
  * Tests for {@link EndpointImplicitUsageProvider}.
- * See http://www.jetbrains.org/intellij/sdk/docs/basics/testing_plugins/testing_highlighting.html
  */
 public class EndpointImplicitUsageProviderTest extends EndpointTestBase {
```

Comment:
I updated this test such that it will now pass on both versions of intellij. Note that it no longer tests the highlighting of java classes - instead it directly tests methods on the `EndpointImplicitUsageProvider` by constructing `PsiElements` from the same test data as before.